HHS/OPRE-OPS
# Adds placeholder for Submitters Notes

## What changed

- adds frontend placeholder for Submitter's Notes

## Issue

#1871 

## How to test

1. Send Agreement 9 for PLANNED TO EXECUTING
2. On Approval page, see placeholder text for Submitters Notes

## Screenshots

<img width="840" alt="image" src="https://github.com/HHS/OPRE-OPS/assets/4629398/2fe9e5b8-6016-4819-9721-e46e06101541">

## Definition of Done Checklist
